### PR TITLE
Use new path for cluster compatible with more modern versions of presto

### PR DESCRIPTION
**Pull request by JamesRTaylor**

### CODE REVIEW

**Comment by JamesRTaylor** [comment]
🔨

**Comment by JamesRTaylor** [comment]
🚀 🙏

File: gateway-ha/src/main/java/com/lyft/data/gateway/ha/clustermonitor/ActiveClusterMonitor.java [hunk 1]
```diff
@@ -80,38 +83,47 @@ public void start() {
...
-      if (responseCode == HttpStatus.SC_OK) {
-        clusterStats.setHealthy(true);
-        BufferedReader reader =
-            new BufferedReader(new InputStreamReader((InputStream) conn.getContent()));
-        StringBuilder sb = new StringBuilder();
-        String line;
-        while ((line = reader.readLine()) != null) {
-          sb.append(line + "\n");
+    // The V1_NODE_PATH is used in 331 while V1_CLUSTER_PATH is used in 318
+    // TODO: Remove V1_CLUSTER_PATH once we're upgraded all clusters.
```
Comment (endoplasmicR) [inline gateway-ha/src/main/java/com/lyft/data/gateway/ha/clustermonitor/ActiveClusterMonitor.java:87]: We don't need to remove the path for V1, instead, we can have a mapping between presto version and the API path
both the `/v1/status` and `v1/info` will provide the Presto version

File: pom.xml [hunk 2]
```diff
@@ -9,7 +9,7 @@
     <artifactId>prestogateway-parent</artifactId>
     <name>prestogateway-parent</name>
     <packaging>pom</packaging>
-    <version>1.6.9</version>
+    <version>1.6.10</version>
```
Comment (endoplasmicR) [inline pom.xml:12]: Nit: can you bump the version to 1.7.0? i think this warrants a minor version bump

File: pom.xml [hunk 3]
```diff
@@ -120,5 +120,16 @@
             </plugin>
         </plugins>
     </build>
-
+  <distributionManagement>
```
Comment (endoplasmicR) [inline pom.xml:123]: We should remove this, this is an open source repo

File: baseapp/pom.xml [hunk 4]
```diff
@@ -7,7 +7,7 @@
     <parent>
         <groupId>com.lyft.data</groupId>
         <artifactId>prestogateway-parent</artifactId>
-        <version>1.6.9</version>
+        <version>1.6.10</version>
```
Comment (endoplasmicR) [inline baseapp/pom.xml:10]: Nit, can you bump the version to 1.7.0? this is a significant change

File: gateway-ha/pom.xml [hunk 5]
```diff
@@ -8,7 +8,7 @@
     <parent>
         <groupId>com.lyft.data</groupId>
         <artifactId>prestogateway-parent</artifactId>
-        <version>1.6.9</version>
+        <version>1.6.10</version>
```
Comment (endoplasmicR) [inline gateway-ha/pom.xml:11]: Nit, can you bump the version to 1.7.0? this is a significant change

File: gateway-ha/src/main/java/com/lyft/data/gateway/ha/clustermonitor/ActiveClusterMonitor.java (hunk 1 above)
Comment (JamesRTaylor) [reply gateway-ha/src/main/java/com/lyft/data/gateway/ha/clustermonitor/ActiveClusterMonitor.java:87]: I'll just get rid of the old path as we're upgrading everything to 332. No need to muck up the code with version dependent switches.

**Comment by endoplasmicR** [comment]
👍

**Pull request merged**